### PR TITLE
feat: New role ExtendedUser

### PR DESCRIPTION
**Pull request by marcometz**
Can see all resources like an admin, but can only edit his own

SVAK-8

Bildschirmfoto 2023-11-16 um 22 38 19
Bildschirmfoto 2023-11-16 um 22 38 27

### CODE REVIEW

File: app/views/accounts/_form.html.erb [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 <div class="form-group">
   <%= f.label :role, "Rolle" %>
-  <%= f.select :role, User.roles.keys.map { |role| [role.humanize, role] }, {}, class: "form-control" %>
+  <%= f.select :role, User.roles.keys.map { |role| [role.camelcase, role] }, {}, class: "form-control" %>
```
Comment (donni106) [inline app/views/accounts/_form.html.erb:26]: this change would be nice being documented in the commit. why was this necessary to change?

File: app/views/accounts/index.html.erb [hunk 2]
```diff
@@ -28,6 +28,14 @@
             Neu erstellte Inhalte sind sofort freigegeben und damit öffentlich lesbar.
           </td>
         </tr>
+        <tr>
+          <td>Extended User</td>
```
Comment (donni106) [inline app/views/accounts/index.html.erb:32]: compared to the other headline, i suggest just `Extended` here

File: app/views/accounts/_form.html.erb (hunk 1 above)
Comment (marcometz) [reply app/views/accounts/_form.html.erb:26]: Screenshot #1 
Naming of "ExtendedUser" in Dropdown would be "Extended user" with `.humanize`

File: app/views/accounts/index.html.erb (hunk 2 above)
Comment (marcometz) [reply app/views/accounts/index.html.erb:32]: Naming of the Role ist "ExtendedUser" als an extension of "User" and is matching the Name in the DropDown and defined by PO in SVAK-8.

File: app/views/accounts/_form.html.erb (hunk 1 above)
Comment (donni106) [reply app/views/accounts/_form.html.erb:26]: ok :/

**Review by donni106 — APPROVED** [review]
